src/components/org/PositionCard/components/PositionInstance.tsx

```diff
@@ -43,7 +47,7 @@ const PositionInstanceComponent: React.FC<PositionInstanceProps> = ({
     const directChildrenTooltipRef = useTooltipRef(`${childCount} positions`, 'above');
     const externalIdTooltipRef = useTooltipRef('External ID: ' + position.externalId, 'below');
 
-    const positionInstanceClasses = classNames(styles.positionInstance, {
+    const positionInstanceClasses = classNames(styles.positionInstance, styles.content, {
```

Comment:
What is styles.content and is it needed?

Reply:
Removed